src/components/common/Toast.tsx

```
@@ -0,0 +1,45 @@
+import ErrorIcon from "@/assets/icons/icon_error.svg";
+import CancelIcon from "@/assets/icons/icon_cancel.svg";
+import { cn } from "@/lib/utils";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)